paper/paper.bib

```diff
@@ -0,0 +1,858 @@
+              breath-hold (BH) tasks is a non-invasive, robust method to
+              estimate cerebrovascular reactivity (CVR). However, movement and
+              breathing-related artefacts caused by the BH can substantially
+              hinder CVR estimates due to their high temporal collinearity with
+              the effect of interest, and attention has to be paid when
+              choosing which analysis model should be applied to the data. In
+              this study, we evaluate the performance of multiple analysis
+              strategies based on lagged general linear models applied on
+              multi-echo BOLD fMRI data, acquired in ten subjects performing a
+              BH task during ten sessions, to obtain subjectspecific CVR and
+              haemodynamic lag estimates. The evaluated approaches range from
+              conventional regression models including drifts and motion
+              timecourses as nuisance regressors applied on singleecho or
+              optimally-combined data, to more complex models including
+              regressors obtained from multi-echo independent component
+              analysis with different grades of orthogonalization in order to
+              preserve the effect of interest, i.e. the CVR. We compare these
+              models in terms of their ability to make signal intensity changes
+              independent from motion, as well as the reliability as measured
+              by voxelwise intraclass correlation coefficients of both CVR and
+              lag maps over time. Our results reveal that a conservative
+              independent component analysis model applied on the
+              optimally-combined multi-echo fMRI signal offers the largest
+              reduction of motion-related effects in the signal, while yielding
+              reliable CVR amplitude and lag estimates, although a conventional
+              regression model applied on the optimally-combined data results
+              in similar estimates. This work demonstrate the usefulness of
+              multi-echo based fMRI acquisitions and independent component
+              analysis denoising for precision mapping of CVR in single
+              subjects based on BH paradigms, fostering its potential as a
+              clinically-viable neuroimaging tool for individual patients. It
+              also proves that the way in which data-driven regressors should
+              be incorporated in the analysis model is not straight-forward due
+              to their complex interaction with the BH-induced BOLD response.
+              \#\#\# Competing Interest Statement The authors have declared no
+              competing interest.",
+  journal  = "bioRxiv",
+  url      = {https://doi.org/10.1101/2020.08.18.256479 },
+  doi      = {10.1101/2020.08.18.256479},
+  month    =  dec,
+  year     =  2020,
+  language = "en"
```

Comment:
```suggestion
  journal = {NeuroImage},
  pages = {117914},
  year = {2021},
  issn = {1053-8119},
  doi = {https://doi.org/10.1016/j.neuroimage.2021.117914},
  url = {https://www.sciencedirect.com/science/article/pii/S1053811921001919},
  language = "en"
```

Reply:
The DOI is valid, so I'm all for it. Congrats!